lib/src/Pos/Diffusion/Full/Block.hs

```diff
@@ -251,85 +215,51 @@ getBlocks logic enqueue nodeId tipHeader checkpoints = do
                            newestHash
         send conv $ mkBlocksRequest lcaChildHash newestHash
         logDebug "Requested blocks, waiting for the response"
-        chainE <- runExceptT (retrieveBlocks conv lcaChild newestHash)
+        chainE <- runExceptT (retrieveBlocks conv numBlocks)
         case chainE of
             Left e -> do
-                let msg = sformat ("Error retrieving blocks frmo "%shortHashF%
+                let msg = sformat ("Error retrieving blocks from "%shortHashF%
                                    " to "%shortHashF%" from peer "%
                                    build%": "%stext)
                                   lcaChildHash newestHash nodeId e
                 logWarning msg
                 throwM $ DialogUnexpected msg
-            Right blocks -> do
-                logDebug $ sformat
-                    ("Retrieved "%int%" blocks of total size "%builder%": "%buildListBounds)
```

Comment:
This message was crucial for debugging. Can you please return it? You can drop the size parameter if you want to.

Reply:
This was a duplicate. It's still logged in `Pos.Block.Network.Retrieval`, which calls this `getBlocks`.